tests/triggers/eks/small/upstream-load.yaml

```diff
@@ -4,7 +4,7 @@ kind: CronJob
 metadata:
   name: eks-small-upstream-load
 spec:
-  schedule: "0 8 * * *"
+  schedule: "0 0 1 * *"
```

Comment:
Are we planning to run this at every month's frequency?

Reply:
Yes, for now. Which is why I moved it to monthly because we want to do only testing with KIT for now.